src/Documentation/glossary.js

```diff
@@ -33,8 +33,8 @@ Initialized by running \`dvc init\` in the **workspace**. It will contain the
       desc: `
 The DVC cache is a hidden storage (by default located in the \`.dvc/cache\`
 directory) for files that are under DVC control, and their different versions.
-For more details, please refer to this
-[document](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory).
+For more details, please refer to this [document]
+(/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory).
```

Comment:
Weird change. Unexpected side-effect of our ESLint config change @algomaster99 @shcheklein ? Happens again twice in the [changes to this file](https://github.com/iterative/dvc.org/pull/715/files#diff-e09d8d350a751ef31fb5badbfa968bf0) below.

Reply:
Ah never mind, these are MD links inside a template literal.